src/components/Button/button.js

```diff
@@ -1,8 +1,8 @@
 import './button.css';
 
-const Button = () => {
+const Button = (props) => {
     return (
-        <button className='button_login'>Login</button>
+        <button className={props.className === true ? 'button_login' : 'button_table'}>{props.children}</button>
```

Comment:
Checking the condition ( props.className === true ) is unnecessary. You can use without "=== true". But, it will be great to change the name of variable from className to isLogin ( boolean )